### PR TITLE
Add documentation about new artifacts parent/child relationship

### PR DESCRIPTION
**Pull request by yannis-rossetto**
This is part of story #18857 define any type of artifact as children

### CODE REVIEW

File: languages/en/user-guide/trackers/usage/artifact-update.rst [hunk 1]
```diff
@@ -164,3 +163,20 @@ Once deactivated:
...
+
+Parent/Child relationship
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before Tuleap 12.5, parent/child relationship was driven by the hierarchy. 
+It meant that the artifact link type ``_is_child`` was automatically set between two artifacts if a hierarchy was defined between the two trackers.
+
+Since Tuleap 12.5, parent/child relationship is only defined by the artifact link type ``_is_child``
+that must be added manually by the user while linking an artifact to another.
+With this feature, any artifact must be defined as a parent/child of another, without any constraints.
```
Comment (vaceletm) [inline languages/en/user-guide/trackers/usage/artifact-update.rst:175]:
```suggestion
With this feature, any artifact can be defined as a child of another, without any constraints. The childs added this way will be displayed everywhere (artifact view, planning view, taskboard, cardwall, etc).
```

File: languages/en/user-guide/trackers/usage/artifact-update.rst [hunk 2]
```diff
@@ -164,3 +163,20 @@ Once deactivated:
...
+
+The hierarchy is now only an helper.
```
Comment (vaceletm) [inline languages/en/user-guide/trackers/usage/artifact-update.rst:177]:
```suggestion
The hierarchy is now only an helper for the Agile Dashboard configuration (what kind of artifacts can be created where, etc).
```

**Review by vaceletm — CHANGES_REQUESTED** [review]

File: languages/en/user-guide/trackers/usage/artifact-update.rst (hunk 2 above)
Comment (yannis-rossetto) [reply languages/en/user-guide/trackers/usage/artifact-update.rst:177]: This not only an helper for AD, but also for tracker triggers (I've listed where this helper is used).

File: languages/en/user-guide/trackers/usage/artifact-update.rst (hunk 1 above)
Comment (yannis-rossetto) [reply languages/en/user-guide/trackers/usage/artifact-update.rst]: I've added this suggestion in a new commit, thanks :+1:

File: languages/en/user-guide/trackers/usage/artifact-update.rst (hunk 2 above)
Comment (vaceletm) [reply languages/en/user-guide/trackers/usage/artifact-update.rst:178]: yup, I forgot, I let you add it

**Pull request merged**